### PR TITLE
Add FBTestManagerTestReporterComposite

### PR DESCRIPTION
**Pull request by plu**
As discussed in #281

### CODE REVIEW

File: XCTestBootstrap/TestManager/FBTestManagerTestReporterComposite.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+#import "FBTestManagerTestReporter.h"
+#import <Foundation/Foundation.h>
+
+@interface FBTestManagerTestReporterComposite : NSObject <FBTestManagerTestReporter>
+
+/**
+ Constructs a Test Reporter with a given List of Other Test Reporters.
+ */
++ (instancetype)withTestReporters:(NSArray<id<FBTestManagerTestReporter>> *)reporters;
```
Comment (lawrencelomax) [inline XCTestBootstrap/TestManager/FBTestManagerTestReporterComposite.h]: Some docs for the args would be good.

File: XCTestBootstrap/TestManager/FBTestManagerTestReporterComposite.h [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "FBTestManagerTestReporter.h"
+#import <Foundation/Foundation.h>
+
```
Comment (lawrencelomax) [inline XCTestBootstrap/TestManager/FBTestManagerTestReporterComposite.h]: You should add `NS_ASSUME_NONNULL_BEGIN` `NS_ASSUME_NONNULL_END`, to expose the nullability contract in the type system

File: XCTestBootstrap/TestManager/FBTestManagerTestReporterComposite.h [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+@interface FBTestManagerTestReporterComposite : NSObject <FBTestManagerTestReporter>
```
Comment (lawrencelomax) [inline XCTestBootstrap/TestManager/FBTestManagerTestReporterComposite.h]: A short comment about what the class is and does.

**Comment by lawrencelomax** [comment]
Looks good from my side, aside from a few comments. I did want to remove `OCMock` as a dependency at some stage so the tests had no external dependencies, but that shouldn't hold up your changes.

**Comment by lawrencelomax** [comment]
Make sure to expose the class as public header in the project and add it to the umbrella header for the Framework.

**Comment by plu** [comment]
Thank you for your feedback! I have addressed everything in separate commits. If you think it looks fine, I can squash all the commits in a single one before you're merging it.

**Comment by lawrencelomax** [comment]
This is really cool 👍

> I can squash all the commits in a single one before you're merging it.

You don't have to worry about this in the future, the facebook robot should do this when I......

@facebook-github-bot shipit

**Comment by facebook-github-bot-4** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1765450550369459/int_phab) to review internal test results.

**Comment by plu** [comment]
Thank you 👍

**Pull request closed without merging**